lambdas/db-provision-user-database/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "rds_security_group" {
+  type = string
+  description = "Security group that allow access to the db cluster"
+}
+
+variable "vpc_id" {
+  type = string
+  description = "The VPC the deployment is in"
+}
+
+variable "prefix" {
+  type = string
+  description = "'prefix' for the deployment ecosystem (Core deployment, data persistence deployment, etc)"
+}
+
+variable "subnet_ids" {
+  type = list(string)
+  description = "Subnets to assign to the database provisioning lambda"
+}
+
+variable "rds_user_password" {
+  description = "Password to set for RDS db user"
+  type = string
+}
+
+variable "rds_access_secret_id" {
```

Comment:
So there are 2 different secrets managed throughout this code, right? One for admin access credentials and one for regular access credentials (to be used by Lamdas writing to RDS, etc). If so, I would recommend naming the one for admin access accordingly, so here `rds_admin_access_secret_id`. And maybe `rds_user_access_secret_id` for the other one?

Reply:
Hmm, yeah, that's a good suggestion, I'll see about sorting that up and down the PR

Reply:
aa0dbf53d